src/Components/Forms/CreateProductForm/CreateProductForm.tsx

```diff
@@ -17,57 +44,65 @@ export const CreateProductForm = () => (
         product_details: '',
         product_price: '',
       }}
-      validationSchema={Yup.object({})}
+      validationSchema={AddProductSchema}
       onSubmit={values => {
-        //should make dispatch here
-        alert(values);
+        let newProduct: IProduct = {
```

Comment:
Usually name on BE an FE should be the same. 
We should skip this mapping. Please rename product props like on BE.

Reply:
Fixed